src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ReadAndUpdateOperation.java

```diff
@@ -181,6 +182,10 @@ private Uni<UpdatedDocument> processUpdate(
                         if (returnDocumentInResponse) {
                           documentToReturn =
                               returnUpdatedDocument ? updatedDocument : originalDocument;
+                          // In this case, we'll apply projection on before/after document (was
+                          // not applied during find() phase
+                          DocumentProjector projector = findOperation().projection();
```

Comment:
Think ReadAndUpdateOperation get the projection in it's constructor? As stated in your comment is it part of different commit?

Reply:
Right, this will not work; it's the identity one.

Reply:
Approved.

Reply:
Discussed offline, yes, this would not work. Commented out and will resolve in follow-up PR; merging to avoid blocking.